### PR TITLE
Allows fullscreen view and improves UI of video button

### PR DESCRIPTION
**Pull request by agbilotia1998**
Fixes: #1875 
Follow up on : #1873

### CODE REVIEW

**Comment by geekyd** [comment]
@agbilotia1998 Please resolved the conflicts. Thanks

**Comment by agbilotia1998** [comment]
@geekyd done! please review.

**Comment by agbilotia1998** [comment]
@mariobehling please check this - 
screenshot-2018-5-23 google io 2017

**Comment by mariobehling** [comment]
Asking again:

> Let's follow best practices more closely as we progress. Please provide screenshots and test links.

**Comment by agbilotia1998** [comment]
Test link: https://agbilotia1998.github.io/IO-videos/

**Comment by mariobehling** [comment]
Thanks. 
1. Please add some padding below the video.
screenshot from 2018-05-28 02-33-11
2. Please change the color of the video icon to match the gray of other areas.
3. Please avoid to show the video button above the text. Show it next to the text.
screenshot from 2018-05-28 02-33-30

**Comment by agbilotia1998** [comment]
@geekyd @mariobehling made the changes. Please review https://github.com/agbilotia1998/IO-videos.git

**Comment by mariobehling** [comment]
Please follow best practices and provide screenshot and deployment link with latest changes.

**Comment by agbilotia1998** [comment]
Screenshot:
screenshot-2018-5-29 google io 2017
The blink of video can be tested here: https://agbilotia1998.github.io/IO-videos/schedule.html

**Comment by geekyd** [comment]
@agbilotia1998 Please resolve conflicts

**Comment by agbilotia1998** [comment]
@geekyd done!

File: src/backend/templates/tracks.hbs [hunk 1]
```diff
@@ -839,15 +841,20 @@
         if(videoURL !== null && videoURL !== '') {
           let isVideoDisplayed = $('#desc-' + div).hasClass('in');
           if (!isVideoDisplayed) {
-            $('#desc-' + div).children('div').prepend('<iframe id = "video-' + div + '" class = "video-iframe col-xs-12 col-sm-12 col-md-12" src="https://www.youtube.com/embed/' + videoURL + '"></iframe>');
+            $('#v' + div).css('display', 'block');
```
Comment (aayusharora) [inline src/backend/templates/tracks.hbs]: Cache `#v + div` in a variable and then use it everywhere

**Comment by agbilotia1998** [comment]
@aayusharora done!! Please review.

**Comment by agbilotia1998** [comment]
@geekyd please review.

**Pull request merged**